jenkins-integration/beaker/install/shared/80_configure_java_args.rb

```diff
@@ -1,38 +1,28 @@
 require 'puppet/gatling/config'
-require 'inifile'
 
 test_name 'Configure java args'
 
 def service_config_name(service_name)
   "/etc/sysconfig/#{service_name}"
 end
 
-def get_service_config(host, service_name)
-  local_service_config_dir = Dir.mktmpdir
-  scp_from(host, service_config_name(service_name), local_service_config_dir)
-  File.join(local_service_config_dir, service_name)
-end
-
-def update_java_args_in_local_config(java_args, local_service_config_path)
-  # Replace the existing JAVA_ARGS in the config
-  ini_file = IniFile.load(local_service_config_path)
-  ini_file['global']['JAVA_ARGS'] = java_args
-  ini_file.save
+def set_bash_variable(host, filepath, variable_name, value)
+  manifest = <<-MANIFEST
+ini_setting { "#{variable_name}":
+  ensure  => present,
+  path    => "#{filepath}",
+  section => "",
+  setting => "#{variable_name}",
+  key_val_separator => "=",
+  value   => "#{value}",
+}
+  MANIFEST
```

Comment:
I tried using the following in my scenario `Jenkinsfile`:

```
server_java_args: "-Xms3g -Xmx3g",
```

That ended up being applied in the /etc/sysconfig/puppetserver file like this:

```
JAVA_ARGS=-Xms3g -Xmx3g
```

The puppetserver service seems to be started with this just fine - including `/usr/bin/java -Xms3g -Xmx3g...` in the command line.  When I run a puppetserver CLI command with that, though, I get this error:

```
[root@k2ziz2dkclswuvi ~]# /opt/puppetlabs/server/bin/puppetserver ruby --version
/etc/sysconfig/puppetserver: line 9: -Xmx3g: command not found
jruby 9.1.9.0 (2.3.3) 2017-05-15 28aa830 OpenJDK 64-Bit Server VM 25.131-b12 on 1.8.0_131-b12 +jit [linux-x86_64]
```

I think it would be good to figure out how to apply double-quotes around the setting value to avoid possible strange behavior in how the variables get used.

I think it would be